src/ert/config/design_matrix.py

```diff
@@ -44,7 +44,7 @@ def from_config_list(cls, config_list: list[str | dict[str, str]]) -> DesignMatr
         filename = Path(cast(str, config_list[0]))
         options = cast(dict[str, str], config_list[1])
         design_sheet = options.get("DESIGN_SHEET", "DesignSheet")
-        default_sheet = options.get("DEFAULT_SHEET", "DefaultSheet")
+        default_sheet = options.get("DEFAULT_SHEET", None)
```

Comment:
So not sure here now. If we specify just the name of XLSL file, by default it does not expect the DEFAULT_SHEET?

Reply:
I sort of agree, but essentially what that means is if there is a default sheet it needs to be specified while design sheet not. And thus we can end up with something like this:
`DESIGN_MATRIX design.xlsl DEFAULT_SHEET:some_name`
which probably is fine